apiserver/appcat/appcat.go

```diff
@@ -1,38 +1,78 @@
 package appcat
 
 import (
-	v12 "appcat-apiserver/apis/appcat/v1"
+	v1 "appcat-apiserver/apis/appcat/v1"
+	"errors"
+	crossplane "github.com/crossplane/crossplane/apis/apiextensions/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	genericregistry "k8s.io/apiserver/pkg/registry/generic"
 	"k8s.io/apiserver/pkg/registry/rest"
 	restbuilder "sigs.k8s.io/apiserver-runtime/pkg/builder/rest"
 	"sigs.k8s.io/apiserver-runtime/pkg/util/loopback"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// +kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch,resourceNames=extension-apiserver-authentication
+// +kubebuilder:rbac:groups="admissionregistration.k8s.io",resources=mutatingwebhookconfigurations;validatingwebhookconfigurations,verbs=get;list;watch
+// +kubebuilder:rbac:groups="",resources=namespaces,verbs=get;list;watch;create;delete;update
+// +kubebuilder:rbac:groups="authorization.k8s.io",resources=subjectaccessreviews,verbs=get;list;watch;create;delete;update
```

Comment:
I don't think you need this permissions. But what you need is permission to access compositions

Reply:
I added all those because I would get errors in the apiserver logs.

Reply:
That's weird.. But I guess :shrug: